pkg/app/src/components/Account/modules/CampaignsSection/CreatedCampaignsSection/campaignCard.tsx

```diff
@@ -30,24 +30,15 @@ const CampaignCard: FC<CampaignCardProps> = ({ campaign }) => {
 			<Box>
 				<CardMedia component="img" src={`${gateway}${getCampaignHeader(campaign)}`} alt="campaign_poster" />
 				<Box sx={{ display: 'flex', justifyContent: 'center' }}>
-					<Card
-						sx={{
-							position: 'relative',
-							width: '6rem',
-							height: '6rem',
-							//TODO: add border radius
-							top: -35,
-							zIndex: 0,
-						}}
-					/>
+					<Card variant="mask" />
 				</Box>
 				<Box sx={{ display: 'flex', justifyContent: 'center' }}>
 					<CardMedia
 						component="img"
 						sx={{
 							width: '4rem',
 							height: '4rem',
-							//TODO: add border radius
+							borderRadius: Number(theme.shape.borderRadius) * 40,
 							top: -115,
```

Comment:
can't you just add a variant to the card component in themes/overrides/card?

Reply:
The problem is that the  card media and several other components doesn't have the variant attribute like stack, container, box,.... and in this case I can't add styles to the card media by adding a variant to the card since the card media doesn't have the variant attribute and styles won't be applied to it

Reply:
but I tried to create a variant for the components that doesn't have a variant attribute and the styles worked and applied the styles but it had an error on the variant attribute
<img width="917" alt="Screen Shot 2022-06-23 at 11 18 56 AM" src="https://user-images.githubusercontent.com/74880499/175251593-8ddca031-7d6f-4193-9685-a63ad9b40e3f.png">

Reply:
yes I understood that that this component doesn't have the variant prop and it is not assignable to type InstriniscAttributes where when using next with typscript we should set the props data types when we're defining our component but this is a mui component I wasn't sure how to add a another prop to it I tried but it didn't work